client/rest-high-level/src/main/java/org/elasticsearch/client/security/user/privileges/Role.java

```diff
@@ -326,11 +326,12 @@ public Role build() {
         public static final String MANAGE_ILM = "manage_ilm";
         public static final String READ_ILM = "read_ilm";
         public static final String MANAGE_ENRICH = "manage_enrich";
+        public static final String ENROLL = "enroll_node";
```

Comment:
```suggestion
        public static final String ENROLL_NODE = "enroll_node";
```

Reply:
Not exposing this privilege to clients, but still documenting it (because otherwise it looks like a backdoor) is also an option that I would support. But that's my bias, your decision.